src/main/java/crazypants/enderio/machine/sagmill/SagMillMachineRecipe.java

```diff
@@ -2,8 +2,10 @@
 
```

Comment:
This already exists in base

and so on, for all recipe stuff. why am I repeating myself? ;)